test/v2-migration-tests/src/test/resources/software/amazon/awssdk/v2migrationtests/maven/before/src/main/java/foo/bar/S3.java

```diff
@@ -146,4 +151,15 @@ private void bucketPrefixArgsMethods(AmazonS3 s3, String bucket, String prefix)
         s3.listObjectsV2(bucket, prefix);
         s3.listVersions(bucket, prefix);
     }
+
+    private void variousMethods(AmazonS3 s3) {
+        s3.deleteVersion("bucket", "key", "versionId");
+        s3.copyObject("sourceBucket", "sourceKey", "destBucket", "destKey");
```

Comment:
How about the there methods getting called with args as CopyObjectRequest ?

```
public CopyObjectResult copyObject(CopyObjectRequest copyObjectRequest)
```


as in v1 we can call s3.copyObject(new CopyObjectRequest().withxxx() ) a

Reply:
Those will be addressed separately later, this PR is for transforms for method overloads with simple type args